docs/svgIcons.md

```diff
@@ -2,25 +2,26 @@ title: SVG Icons
 
 # SVG Icons in Monster-UI
 Monster UI includes 4 SVG icon collections:
+
 1. Google Drive Logos
```

Comment:
I'd argue this list does not need to be ordered. In fact, it should not be a list but rather titles:

```suggestion
##### Google Drive Logos
```